docs/_newsfragments/648.rst

```diff
@@ -0,0 +1 @@
+Add ``path`` field converted to match all paths that start with a prefix.
```

Comment:
In newsfragments, IMHO we should tell a story, not phrase in the imperative as docstrings.
Also, _converted_ (sic) :arrow_right: _converter_.

Reply:
tried rewording. I'm afraid I'm not a story teller, so feel free to suggest an alternative